node/network/availability-distribution/src/lib.rs

```diff
@@ -681,7 +770,14 @@ impl AvailabilityDistributionSubsystem {
 	where
 		Context: SubsystemContext<Message = AvailabilityDistributionMessage>,
 	{
-		let mut state = ProtocolState::default();
+		let mut state = ProtocolState {
```

Comment:
Not sure what we gain with the explicit initialization of all members?

Reply:
I removed the `Default` because `JaegerSpan` is not `Default`